tests/cupyx_tests/scipy_tests/special_tests/test_log_softmax.py

```diff
@@ -0,0 +1,49 @@
+import cupy
+from cupy import testing
+import cupyx.scipy.special  # NOQA
+
+import scipy.special  # NOQA
+
+
+atol = {'default': 1e-6, cupy.float64: 1e-14}
+rtol = {'default': 1e-6, cupy.float64: 1e-14}
```

Comment:
You may add `cupy.float16: 1e-4` for float16, which generally has very low precision. (The value `1e-4` may vary. Please adjust it.)

Reply:
Have updated the PR according to our offline discussions. Thanks a ton for your input and time :)